### DISCUSSION TITLE
Documentation and Quick Start links - Access Denied

### DISCUSSION THREAD

**Discussion by mooragor** [reqnroll/Reqnroll#102] [Q&A] [answered]
Hi there, I was trying to have a read through the migration guide  and have noticed that a lot of the documentation links on the main website are 'Access Denied' - any thoughts?

https://go.reqnroll.net/guide-migrating-from-specflow
https://docs.reqnroll.net/
https://go.reqnroll.net/quickstart

**Comment by gasparnagy** [comment, marked as answer]
@mooragor I think your problem is the same as described in #79 . For some unknown reason, some internet providers block our documentation site.

Could you please:
- Confirm that https://reqnroll.net works?
- Try the same URLs from another provider (e.g. from your phone)
- Try to contact the admins of your internet provider service to get information WHY it is blocked. It is very frustrating that some people get these errors, but we have no idea why...

I close this question here, let's continue the discussion at #79.
